alacritty/src/event.rs

```diff
@@ -676,7 +676,12 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
             HintAction::Command(command) => {
                 let text = self.terminal.bounds_to_string(*hint.bounds.start(), *hint.bounds.end());
                 let mut args = command.args().to_vec();
-                args.push(text);
+
+                match text.strip_suffix('\n') {
+                    Some(text) => args.push(text.to_string()),
+                    None => args.push(text),
+                }
```

Comment:
```
let text = text.strip_suffix('\n').map(str::to_owned).unwrap_or(text);
```

There might be an even easier way to do it, but that's the best I could come up with.